README.md

```diff
@@ -42,6 +42,9 @@ Simple Example
 
 ```cpp
 #include <allegro_flare/allegro_flare.h>
+using namespace allegro_flare;
+using namespace AllegroFlare;
```

Comment:
I actually tried compiling this example program on its own last week, and it actually doesn't even compile. 😂   I [added it as an example program here](https://github.com/allegroflare/allegro_flare/blob/master/examples/simple_example.cpp), but have not included it in the `README.md`, (and at the moment I can't recall if the example program currently builds or not).

Also, it would be nice if there were a way to dynamically embed the example program in the `README.md`.  I looked around a bit and the only thing I could find was a user who created a tool that manually updated markdowns with a specific syntax.  https://github.com/isaacs/github/issues/471.  It might make a good commit hook, but that's for another time.